2018-12-27 11:22:10.345 [error] <0.377.0>@Elixir.WorkersManager:handle_cast:21 gen_server <0.377.0> terminated with reason: bad arithmetic expression in 'Elixir.WorkersManager':handle_cast/2 line 21
2018-12-27 11:22:10.356 [error] <0.377.0>@Elixir.WorkersManager:handle_cast:21 CRASH REPORT Process <0.377.0> with 2 neighbours crashed with reason: bad arithmetic expression in 'Elixir.WorkersManager':handle_cast/2 line 21
2018-12-27 16:40:08.048 [error] <0.361.0> Supervisor 'Elixir.ElixirTryout.Supervisor' had child 'Elixir.DrainStop' started with 'Elixir.DrainStop':start_link([{timeout,10000},{endpoint,'Elixir.ElixirTryoutWeb.Endpoint'}]) at <0.497.0> exit with reason killed in context shutdown_error
